Linked list with void pointers.c

```diff
@@ -5,29 +5,26 @@ typedef struct node
 {
     void *data;
     struct node *next;
+    int (*funptr)();
 } node;
-struct node *head = NULL;
-struct node *tail = NULL;
-void insert(node **head, node **tail, const void * pointer, size_t size)
+typedef struct note

```

Comment:
what is Note ? Maybe ListHead can be good name

Reply:
ok annayha